pkg/fleet/internal/cdn/cdn.go

```diff
@@ -50,87 +46,110 @@ type orderConfig struct {
 }
 
 // New creates a new CDN.
-func New(env *env.Env) *CDN {
-	return &CDN{
-		env: env,
+func New(env *env.Env) (*CDN, error) {
+	config := pkgconfigsetup.Datadog()
+	detectenv.DetectFeatures(config)
+	client, err := remoteconfig.NewHTTPClient(
+		config,
+		fmt.Sprintf("https://remote-config.%s", env.Site),
+		fmt.Sprintf("https://config.%s", env.Site),
```

Comment:
Maybe not for this PR but I guess we should/could make this customisable with a URL rather than a site